### PR TITLE
tidy

### PR DESCRIPTION
**Pull request by richardiphillips**

### CODE REVIEW

File: tests/Integration/Service.Tests/LabelPrintModuleSpecs/WhenSearchingSuppliers.cs [hunk 1]
```diff
@@ -1,22 +1,29 @@
 ﻿namespace Linn.Production.Service.Tests.LabelPrintModuleSpecs
 {
+    using System.Collections.Generic;
+    using System.Linq;
+
```
Comment (iaincrawf) [inline tests/Integration/Service.Tests/LabelPrintModuleSpecs/WhenSearchingSuppliers.cs]: Was this all with an automated tool or something? Looks good but these spaces between the usings seem a bit strange to me

File: tests/Integration/Service.Tests/LabelTypesModuleSpecs/WhenGettingAll.cs [hunk 2]
```diff
@@ -1,15 +1,20 @@
 ﻿namespace Linn.Production.Service.Tests.LabelTypesModuleSpecs
 {
+    using System.Collections.Generic;
+    using System.Linq;
+
```
Comment (iaincrawf) [inline tests/Integration/Service.Tests/LabelTypesModuleSpecs/WhenGettingAll.cs]: same with the extra lines

File: tests/Integration/Service.Tests/ManufacturingOperationsModuleSpecs/WhenGettingAll.cs [hunk 3]
```diff
@@ -1,15 +1,20 @@
 ﻿namespace Linn.Production.Service.Tests.ManufacturingOperationsModuleSpecs
 {
+    using System.Collections.Generic;
+    using System.Linq;
+
```
Comment (iaincrawf) [inline tests/Integration/Service.Tests/ManufacturingOperationsModuleSpecs/WhenGettingAll.cs]: same with lines but probs doesn't really matter

Comment (richardiphillips) [reply tests/Integration/Service.Tests/ManufacturingOperationsModuleSpecs/WhenGettingAll.cs]: yeah. resharper done it. I just believed it.

**Pull request merged**